newIDE/app/src/EventsFunctionsExtensionEditor/OptionsEditorDialog/ExtensionOptionsEditor.js

```diff
@@ -217,16 +218,15 @@ export const ExtensionOptionsEditor = ({
               forceUpdate();
             }}
           />
-          <TextField
-            floatingLabelText={
-              <Trans>Author (Name, email or GitHub handle)</Trans>
+          <UsersAutocomplete
+            userIds={eventsFunctionsExtension.getAuthorIds()}
+            floatingLabelText={<Trans>Authors</Trans>}
+            helperText={
+              <Trans>
+                Select all the extensions contributors usernames. They will be
```

Comment:
Not sure about this but, is it more correct to say:`Select the usernames of the persons that contributed to this extension` ?

Reply:
yup, you're right :+1: